cypress-tests/cypress/support/commands.js

```diff
@@ -49,7 +53,7 @@ Cypress.Commands.add("merchantCreateCallTest", (merchantCreateBody, globalState)
 
     const xRequestId = response.headers['x-request-id'];
     if (xRequestId) {
-      cy.task('cli_log', "x-request-id ->> " + xRequestId);
+      cy.task('cli_log', "x-request-id -> " + xRequestId);
     } else {
       cy.task('cli_log', "x-request-id is not available in the response headers");
     }
```

Comment:
we could use reuse the func  `logRequestId` ryt?

Reply:
Missed this, updated.